src/ui/issuecolumn/ColumnControl.java

```diff
@@ -229,8 +229,19 @@ public void setCurrentlyDraggedColumnIndex(int i) {
 	
 	public void closeCurrentColumn() {
 		if (currentlySelectedColumn.isPresent()) {
-			closeColumn(currentlySelectedColumn.get());
-			currentlySelectedColumn = Optional.empty();
+			int columnIndex = currentlySelectedColumn.get();
+			System.out.println("Going to close column " + columnIndex);
+			closeColumn(columnIndex);
+			if (getChildren() == null || getChildren().size() == 0) {
```

Comment:
I don't think `getChildren()` will ever be null. This can be safely omitted, or made into an assertion.